scm-info/com/google/_artifact/google/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://github.com/google/google-maven-parents.git"
+tagMapping:
+  - pattern: (.*)
```

Comment:
This should not be necessary. The heuristic should be able to match simple patterns like this.